File: au-association-go/model/system/ActivityComplete.go
package system

import "time"

type Model struct {
	ID       uint      `gorm:"column:id;primary_key;AUTO_INCREMENT"`
	CreateAt time.Time `gorm:"column:create_time"`
	UpdateAt time.Time `gorm:"column:update_time"`
}

type ActivityComplete struct {
	Model
	ActivityName string    `json:"activityName"`
	Twitter      string    `json:"twitter"`
	Date         time.Time `json:"date"`
	File1        string    `json:"file1"`
	File2        string    `json:"file2"`
	Description  string    `json:"description"`
	Ass          int       `json:"ass"`
	AssStruct    Ass       `gorm:"foreignKey:ass;references:assid"`
}

func (ActivityComplete) TableName() string {
	return "activity_complete"
}

func (ActivityComplete) GetInfo(info ActivityComplete) (result map[string]interface{}) {
	result = make(map[string]interface{})
	result["id"] = info.ID
	result["activityName"] = info.ActivityName
	result["date"] = info.Date
	result["twitter"] = info.Twitter
	result["file1"] = info.File1
	result["file2"] = info.File2
	result["description"] = info.Description
	//result["assName"] = info.AssStruct.Assname
	//result["assId"] = info.AssStruct.Assid
	result["createTime"] = info.CreateAt
	result["updateTime"] = info.UpdateAt
	return
}

File: au-java/src/main/java/com/fran6k/study/springbootstudy/bean/InReviewBean/ActivityComplete.java
package com.fran6k.study.springbootstudy.bean.InReviewBean;

import com.alibaba.fastjson.JSONObject;
import com.fran6k.study.springbootstudy.bean.Ass;
import lombok.Data;
import net.sf.json.JSONArray;
import org.springframework.data.annotation.CreatedDate;
import org.springframework.data.annotation.LastModifiedDate;
import org.springframework.data.jpa.domain.support.AuditingEntityListener;

import javax.persistence.*;
import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.List;

@Entity
@Table(name = "activity_complete")
@Data
@EntityListeners(AuditingEntityListener.class)
public class ActivityComplete {
    @Id
    @GeneratedValue(strategy= GenerationType.AUTO)
    private int id;
    private String activityName;
    private String twitter;
    private Date date;
    private String file1;
    private String file2;
    private String description;
    @CreatedDate//自动添加创建时间的注解
    private Date createTime;
    @LastModifiedDate//自动添加更新时间的注解
    private Date updateTime;

    @ManyToOne(targetEntity= Ass.class)
    @JoinColumn(name="ass",referencedColumnName = "assid")
    private Ass ass;

    public JSONObject getInfo(){
        String strDateFormat = "yyyy-MM-dd HH:mm:ss";
        SimpleDateFormat sdf = new SimpleDateFormat(strDateFormat);

        JSONObject jsonObject = new JSONObject();
        jsonObject.put("id",id);
        jsonObject.put("activityName",activityName);
        jsonObject.put("date",sdf.format(date));
        jsonObject.put("twitter",twitter);
        jsonObject.put("file1",file1);
        jsonObject.put("file2",file2);
        jsonObject.put("description",description);
        jsonObject.put("assName",ass.getAssname());
        jsonObject.put("assId",ass.getAssid());
        jsonObject.put("createTime",sdf.format(createTime));
        jsonObject.put("updateTime",sdf.format(updateTime));
        return jsonObject;
    }
}
